wherehows-backend/app/actors/EtlJobActor.java

```diff
@@ -51,7 +51,7 @@
 
   @Override
   public void onReceive(Object message) throws Exception {
-    final String configDir = ETL_TEMP_DIR + "/exec";
+    String configDir = ETL_TEMP_DIR;
```

Comment:
We can just set configDir = ETL_TTEMP_DIR + "/" + msg.getEtlJobName() + "/" + msg.getWhEtlExecId() here instead?

Reply:
We can't set the directory structure here as we are constructing the etljob properties(msg object) only after this line.